tests/dart/DartLintBearTest.py

```diff
@@ -30,3 +37,23 @@
                                      valid_files=(good_file,),
                                      invalid_files=(bad_file,),
                                      tempfile_kwargs={"suffix": ".dart"})
+
+
+@skipIf(which('dartanalyzer') is None, 'dartanalyzer is not installed')
+class DartLintBearConfigTest(LocalBearTestHelper):
+
+    def test_config_failure_use_spaces(self):
+        section = Section("name")
+        section.append(Setting('use_spaces', False))
+        bear = DartLintBear(section, Queue())
+
+        with pytest.raises(AssertionError):
```

Comment:
Well, since py.test is used as a test runner, I thought it best to use it directly as much as possible. But if it serves the uniformity of the code base, I'll change it ;)